__tests__/components/Tabbar/Tabbar.test.js

```diff
@@ -28,20 +29,4 @@ describe('Tabbar should run correctly', () => {
     );
     expect(toJSON()).toMatchSnapshot();
   });
-
-  it('tabbar onPress should run correctly', async () => {
-    const navigation = {
-      emit: jest.fn().mockImplementation(() => ({
-        defaultPrevented: false
-      })),
-      navigate: jest.fn()
-    };
-    const {getByTestId} = render(
-      <Tabbar state={state} navigation={navigation} descriptors={descriptors} />
-    );
-    await fireEvent.press(getByTestId('btn1'));
-    expect(navigation.emit).toHaveBeenCalled();
-    await fireEvent.press(getByTestId('btn0'));
-    expect(navigation.navigate).toHaveBeenCalled();
-  });
 });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case for the 'tabbar onPress' event has been removed. This might lead to less coverage in your tests. If the functionality is still relevant, consider adding it back.
```diff
+  it('tabbar onPress should run correctly', async () => {
+    const navigation = {
+      emit: jest.fn().mockImplementation(() => ({
+        defaultPrevented: false
+      })),
+      navigate: jest.fn()
+    };
+    const {getByTestId} = render(
+      <Tabbar state={state} navigation={navigation} descriptors={descriptors} />
+    );
+    await fireEvent.press(getByTestId('btn1'));
+    expect(navigation.emit).toHaveBeenCalled();
+    await fireEvent.press(getByTestId('btn0'));
+    expect(navigation.navigate).toHaveBeenCalled();
+  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->